Data/models/TabMO.swift

```diff
@@ -134,11 +134,14 @@ public final class TabMO: NSManagedObject, CRUD {
         deleteAll(context: .new(inMemory: false))
     }
     
+    public class func deleteAllNormalTabs() {
+        deleteAll(predicate: NSPredicate(format: "isPrivate != true"), context: .new(inMemory: false))
```

Comment:
We do not save private tabs anymore, you can just call `deleteAll()` context is also set to `.new` by default

Reply:
Fixed.